### DISCUSSION TITLE
Gifox

### DISCUSSION THREAD

**Discussion by mrleblanc101** [elrumo/macOS_Big_Sur_icons_replacements#1468] [Icon request] [answered]
https://gifox.app/

**Comment by itjustcrashed** [comment, marked as answer]
Waiting approval but here's the PNG:
![Gifox](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/151413500/685c20b3-5f25-4e1f-a472-69080be040c3)

**Reply by mrleblanc101** [reply]
Thanks

**Reply by itjustcrashed** [reply]
Your welcome! [My profile](https://macosicons.com/#/u/wannafedora)

**Comment by itjustcrashed** [comment]
Waiting for approval but here's the PNG:
![Gifox](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/151413500/7a1d7527-3c81-4359-aea1-142d638b000e)
